lib/matplotlib/backends/backend_ps.py

```diff
@@ -1092,14 +1094,14 @@ def print_figure_impl(fh):
                     int(source_date_epoch)).strftime("%a %b %d %H:%M:%S %Y")
             else:
                 source_date = time.ctime()
-            print("%%CreationDate: " + source_date, file=fh)
-            print("%%Orientation: " + orientation, file=fh)
-            if not isEPSF:
-                print("%%DocumentPaperSizes: "+papertype, file=fh)
-            print("%%%%BoundingBox: %d %d %d %d" % bbox, file=fh)
-            if not isEPSF:
-                print("%%Pages: 1", file=fh)
-            print("%%EndComments", file=fh)
+            print("%%Creator: {creator_str}\n"
+                  "%%CreationDate: {source_date}\n"
+                  "%%Orientation: {orientation}\n"
+                  "%%BoundingBox: {bbox[0]} {bbox[1]} {bbox[2]} {bbox[3]}\n"
+                  "%%EndComments\n"
+                  .format(creator_str=creator_str, source_date=source_date,
+                          orientation=orientation, bbox=bbox),
+                  end="", file=fh)
```

Comment:
Could just spell it `fh.write` at this point

Reply:
Then I'd rather change the whole thing to use fh.write throughout, but... too lazy.